pkg/selectors/kernel.go

```diff
@@ -489,6 +524,11 @@ func ParseMatchArg(k *KernelSelectorState, arg *v1alpha1.ArgSelector, sig []v1al
 		if err != nil {
 			return fmt.Errorf("writeMatchValuesInMap error: %w", err)
 		}
+	case SelectorOpSport, SelectorOpDport:
+		err := writeMatchRangesInMap(k, arg.Values, argTypeU64)
```

Comment:
shouldn't `argTypeU64` be `ty` as do above?

Reply:
No because ty would be sock or skb. As these are structs, we force the type based on the operator.